spec/models/submission_spec.rb

```diff
@@ -16,7 +16,7 @@
     expect(described_class.create.errors[:user_type]).to be_present
   end
 
-  [:download, :render, :run, :test].each do |action|
```

Comment:
You are right, it should not be deleted here. I need to solve problem with this test in other way.